coruscant/src/components/ProjectHeader.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { Paper } from 'material-ui'
+import Healthy from 'material-ui/svg-icons/action/check-circle'
+import Warning from 'material-ui/svg-icons/content/report'
+import Failing from 'material-ui/svg-icons/navigation/cancel'
+
+import IconMenu from 'material-ui/IconMenu'
+import MenuItem from 'material-ui/MenuItem'
+import IconButton from 'material-ui/IconButton'
+import MoreHorizIcon from 'material-ui/svg-icons/navigation/more-horiz'
+import { yellow500, red500, green500 } from 'material-ui/styles/colors'
+import Theme from '../../style/theme'
+import { register } from './../actions/session'
+
+
+const styles = {
+  header: {
+    width: '90%',
+    textAlign: 'center',
+    display: 'inline-block',
+    padding: 10
+  },
+  headerContainer: {
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  ProjectNameFont: {
+    fontFamily: 'Arial',
```

Comment:
I think it's better to standardize around 1 font, and Roboto is pretty much what we are using everywhere since it's Material Design

Reply:
Yeah, you can just leave out `fontFamily` and it will use the default defined in the theme.